libethereum/Client.cpp

```diff
@@ -828,6 +828,7 @@ void Client::rewind(unsigned _n)
     auto h = m_host.lock();
     if (h)
         h->reset();
+    m_tq.clear();
```

Comment:
Is this change related to the other changes?

Reply:
if there are any transactions in the queue. with error status or not. it should be removed from the queue after calling the rewind method.

Reply:
the same for `m_bq` then, too?

Reply:
if this wont affect blockchain memory of which blocks could potentially be valid, then I think yes
m_bq is a client/networking variable, right?

Reply:
So `m_bq` was added below this line.